Python/optimizer_analysis.c

```diff
@@ -489,6 +486,65 @@ optimize_uops(
 
 }
 
+static inline bool
+op_is_simple_load(int opcode) {
```

Comment:
Can we avoid mixing refactorings and adding new features?

`remove_unneeded_uops` coould do with cleaning up, but it should be done in another PR.